solr/contrib/ltr/example/techproducts-features.json

```diff
@@ -1,26 +1,26 @@
 [
 {
     "name":  "isInStock",
-    "type": "org.apache.solr.ltr.feature.impl.FieldValueFeature",
+    "class": "org.apache.solr.ltr.feature.impl.FieldValueFeature",
```

Comment:
Can you grep for any other .json files which we need to change type=> class which we may have missed?